tools/java_api/java_test/QueryResultTest.java

```diff
@@ -0,0 +1,204 @@
+package tools.java_api.java_test;
+
+import tools.java_api.*;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.BeforeAll;
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.AfterAll;
+
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+
+public class QueryResultTest {
+    private static  KuzuDatabase db;
+    private static KuzuConnection conn;
+
+    @BeforeAll
+    static void getDBandConn() {
```

Comment:
`getDBandConn` and `destroyDBandConn` appears in multiple test files. This can be refactored. Try to create a base class for testing and have all your test classes inherit it.